### ISSUE TITLE
Not with Reshade on

### ISSUE THREAD

**Issue by Allister-Vinris** [Lyall/SoulHackers2Fix#4]
Your fix is great and came faster than a dev patch, so thanks. However, it does not seem to work together with reshade, it either crashes the game or turn the screen black as long as any effect is applied.

My reshading was subtle so I can do without, but I thought you should know.

**Comment by Lyall** [comment]
It seems like it's linked to the custom resolution patch. I've been meaning to redo this as it doesn't quite work the way I want it to yet and if it breaks Reshade that's even more reason to rework it! For now though if you want to use Reshade then I'd recommend setting `CustomResolution = false` in your config for just now and dealing with the game's resolution changing shenanigans until I can fix up a new version of the patch.

**Comment by Allister-Vinris** [comment]
Alright, thank you ! I'll try that tomorrow. good job !

Scratch that, I tried right now and it worked just fine !

**Comment by Lyall** [comment]
So I've been banging my head on this for the past few days and currently I don't see a way to make the custom resolution patch work with Reshade I'm afraid. The game does not allow for exclusive fullscreen at any other aspect ratio than 16:9 and frankly the game does some very weird things with resolution on startup. As soon as I mess with those methods it seems to break Reshade. I think all I can hope for on this front is either a game patch or a BepInEx update, depending on which is causing issues (it's likely the former).

I'll leave this issue open for now until/if anything changes.

TL;DR: No custom resolution support with Reshade, disable it using `CustomResolution = false` in your config file if you want to use Reshade.

**Comment by Allister-Vinris** [comment]
Oh, well I suppose that's a shame. 
I mean I only have a 4k 16:9, I wanted the fix for the sliding control issue at over 60 fps, and with custom resolution at false it works as intended so, you know, there is that at least. 
Plus the game looks pretty good even without reshade so don't bang your head to hard okay ? You've done quite the fantastic job already.
